webpack.config.js

```diff
@@ -8,6 +8,15 @@ module.exports = {
         use: {
           loader: "babel-loader"
         }
+      },
+      {
+        test: /\.(png|jpe?g|gif)$/,
+        use: [
+          {
+            loader: "file-loader",
+            options: {}
+          }
+        ]
```

Comment:
Looks good. Definitely could refactor some things. But, I think for most of it, it's best to use your time to move on with the project and come back to refactoring when you have more time. Good work!